cmd/podman/common.go

```diff
@@ -434,6 +434,10 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"read-only", false,
 		"Make containers root filesystem read-only",
 	)
+	createFlags.Bool(
+		"read-only-tmpfs", true,
+		"When running containers in read-only mode mount tmpfs on /run, /tmp and /var/tmp",
```

Comment:
mount tmpfs on /run, /tmp and /var/tmp as read-only ?

Reply:
```suggestion
		"When running containers in read-only mode mount a read-write tmpfs on /run, /tmp and /var/tmp",
```